src/api/journalEntries/journalEntry.ts

```diff
@@ -30,7 +30,7 @@ export const useJournalEntry = (id: string) => {
   });
 };
 
-const createJournalEntryQuery = `
+const createJournalEntryMutation = gql`
```

Comment:
Definitely better naming for the queries, the function that does the mutating, and the hooks